include/deal.II/base/mpi_remote_point_evaluation.h

```diff
@@ -156,6 +156,18 @@ namespace Utilities
       bool
       is_map_unique() const;
 
+      /**
+       * Return if all points could be associated to any process.
```

Comment:
With reference to issue https://github.com/dealii/dealii/issues/13500, I would prefer that the nomenclature used here for documenting the function is not MPI-style ("associated to any process"), but rather from a serial perspective (understanding the support for parallel computations a detail of the implementation). The `reinit()` function receives tria and mapping, which should be referenced here, i.e. the documentation of this function should explain what "found" means. In addition, the documentation of `reinit()` should be extended.